src/views/forms/TradeForm/PlaceOrderButtonAndReceipt.tsx

```diff
@@ -79,6 +80,17 @@ export const PlaceOrderButtonAndReceipt = ({
       label: stringGetter({ key: STRING_KEYS.FEE }),
       value: <Output type={OutputType.Fiat} value={fee} useGrouping />,
     },
+    {
+      key: 'max-reward',
+      label: (
+        <>
+          {stringGetter({ key: STRING_KEYS.MAXIMUM_REWARDS })}
+          {/* <AssetIcon symbol="DYDX" /> */}
+        </>
+      ),
+      value: <Output type={OutputType.Asset} value={reward} useGrouping />,
```

Comment:
Would it be worth adding a `tag` for `dv4tnt`

Reply:
it's not in the design, but could make sense in the mean time